src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
+    enum Color {
```

Comment:
enum and inner classes should be placed in the end of the outer class

Reply:
not fixed